src/agents/handoffs.py

```diff
@@ -49,8 +49,24 @@ class HandoffInputData:
     handoff and the tool output message representing the response from the handoff output.
     """
 
+    run_context: RunContextWrapper[Any] | None = None
+    """
+    The run context at the time the handoff was invoked.
+    Note that, since this property was added later on, it's optional for backwards compatibility.
+    """
+
+    def clone(self, **kwargs: Any) -> HandoffInputData:
```

Comment:
added clone method for convenience but if we don't want to have this, happy to revert this change